build/packaging/containers/shared/yumalldeps.sh

```diff
@@ -48,7 +48,9 @@ mylog() {
 
 getdep() {
     tmp_all="$(yum deplist $YUM_OPTIONS $* | awk '/provider/ {print $2}' | sort -u )"
-    remaining="$(echo "$tmp_all" | grep $PKG_FAMILY | sed ':a;N;$!ba;s/\n/ /g' )"
+    # Printing all packages that can provide the requirement may result in including in the list multiple providers
+    # incompatible with each others. An install command will result in error unless "--skip-broken" is added
```

Comment:
typo: `other` (instead of `others`)

Reply:
fixed, using one another (there may be more than 2)